### PR TITLE
task: update bundle to v24.10.0-beta.1

### PR DESCRIPTION
**Pull request by nvidia-ci-cd**
Created by the *ocp-bundle* job in [Mellanox/network-operator OCP bundle CI](https://github.com/Mellanox/network-operator/actions/runs/10921479565).

### CODE REVIEW

**Comment by rollandf** [comment]
@maze88 we are missing `signed-off` in the commit message as:

`Signed-off-by: nvidia-ci-cd <nvidia-ci-cd@nvidia.com>`

**Comment by maze88** [comment]
> @maze88 we are missing `signed-off` in the commit message as:
> 
> `Signed-off-by: nvidia-ci-cd <nvidia-ci-cd@nvidia.com>`

I _can_ add it, but:
- The repo settings _do not_ demand this.
- I believe signing off is supposed to be a conscient and manual action...

Are you sure I should add signing off to the automation?

**Comment by rollandf** [comment]
> > @maze88 we are missing `signed-off` in the commit message as:
> > `Signed-off-by: nvidia-ci-cd <nvidia-ci-cd@nvidia.com>`
> 
> I _can_ add it, but:
> 
> * The repo settings _do not_ demand this.
> * I believe signing off is supposed to be a conscient and manual action...
> 
> Are you sure I should add signing off to the automation?

We do it here:

https://github.com/Mellanox/network-operator/pull/1071/commits/cf87d132761a9aeb0280232ea7a95d9e04bc7ddc

File: bundle/metadata/annotations.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: nvidia-network-operator
-  operators.operatorframework.io.bundle.channels.v1: v24.7.0-rc.1,stable
-  operators.operatorframework.io.bundle.channel.default.v1: v24.7.0-rc.1
+  operators.operatorframework.io.bundle.channels.v1: stable,v24.10.0-beta
```
[the remark this answers is not in the document]
Comment (maze88) [reply bundle/metadata/annotations.yaml:7]: affirmative..!
fixing in https://github.com/Mellanox/network-operator/pull/1076

**Comment by rollandf** [comment]
Closing this one.
Let see next beta how it goes

**Pull request closed without merging**